src/main/java/kg/jarkyn/server/Server.java

```diff
@@ -0,0 +1,33 @@
+package kg.jarkyn.server;
+
+import kg.jarkyn.server.Incoming.Requester;
+import kg.jarkyn.server.Outgoing.Responder;
+import kg.jarkyn.server.Outgoing.Response;
+
+public class Server {
+    private final ReceivingSocket serverSocket;
+    private final Responder responder;
+
+    public Server(ReceivingSocket serverSocket, Responder responder) {
```

Comment:
I see your point.
I have a conflict with java.net package, need something that can reflect that this is a server side socket yet distinct from java.net.ServerSocket, will need to think about this more

will have give it